src/main/java/uk/gov/companieshouse/overseasentitiesapi/exception/HashingException.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.companieshouse.overseasentitiesapi.exception;
+
+public class HashingException extends RuntimeException {
```

Comment:
Should this be a ServiceException?

Reply:
Do you think wrap it in service exception 
} catch (NoSuchAlgorithmException e) {
    throw new ServiceException("Hashing failed", new HashingException("Cannot encode Managing Officer ID", e));

or replace it altogether??

Reply:
I'd just throw ServiceException directly